dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_document_path
+
+
+def upload_file(bucket, dry_run, file_path, framework_slug, bucket_category,
+                document_category, document_type):
+    supplier_id = get_supplier_id_from_framework_file_path(file_path)
+    document_name = '{0}.{1}'.format(document_category, document_type)
+    upload_path = get_document_path(framework_slug, supplier_id, bucket_category,
+                                    document_name)
+    if not dry_run:
+        with open(file_path) as file_contents:
+            bucket.save(upload_path, file_contents, acl='private', move_prefix=None)
```

Comment:
We should probably set `timestamp` here (like in https://github.com/alphagov/digitalmarketplace-scripts/blob/master/scripts/oneoff/set-agreements-content-disposition.py#L32, except using the current time), so that if files are modified we can keep the original upload date.